packages/common/src/core/schemas/internal/follow/FollowCardUiSchema.ts

```diff
@@ -0,0 +1,230 @@
+import { IArcGISContext } from "../../../../ArcGISContext";
+import { CardEditorOptions } from "../EditorOptions";
+import { IUiSchema } from "../../types";
+
+/**
+ * @private
+ * Exports the uiSchema of the stat card
+ * @returns
+ */
+export const buildUiSchema = (
+  i18nScope: string,
+  config: CardEditorOptions,
+  context: IArcGISContext
+): IUiSchema => {
+  return {
+    type: "Layout",
+    elements: [
+      {
+        label: "Select Entity to Follow",
+        scope: "/properties/followedItemId",
+        type: "Control",
+        options: {
+          control: "hub-field-input-gallery-picker",
+          targetEntity: "item",
+          catalogs: [
+            {
+              schemaVersion: 1,
+              title: "Esri",
+              scopes: {
+                item: {
+                  targetEntity: "item",
+                  filters: [
+                    {
+                      predicates: [
+                        {
+                          type: "Hub Project",
+                        },
+                      ],
+                    },
+                  ],
+                },
+              },
+            },
+          ],
+          facets: [
+            {
+              label: "Access",
+              key: "access",
+              display: "multi-select",
+              operation: "OR",
+              options: [
+                {
+                  label: "Public",
+                  key: "public",
+                  selected: false,
+                  predicates: [
+                    {
+                      access: "public",
+                    },
+                  ],
+                },
+                {
+                  label: "Shared",
+                  key: "shared",
+                  selected: false,
+                  predicates: [
+                    {
+                      access: "shared",
+                    },
+                  ],
+                },
+                {
+                  label: "Org",
+                  key: "org",
+                  selected: false,
+                  predicates: [
+                    {
+                      access: "org",
+                    },
+                  ],
+                },
+                {
+                  label: "Mine",
+                  key: "mine",
+                  selected: false,
+                  predicates: [
+                    {
+                      owner: "paige_pa",
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              label: "Types",
```

Comment:
Since this is limited to Sites, and in the future we'll add Projects & Initiatives... maybe remove the Types facet for now?